tests/unit_tests/test_wave_06.py

```diff
@@ -109,16 +115,26 @@ def test_swap_best_by_category_reordered():
         their_priority="Decor"
     )
 
-    raise Exception("Complete this test according to comments below.")
+    # raise Exception("Complete this test according to comments below.")
     # *********************************************************************
     # ****** Complete Assert Portion of this test **********
     # *********************************************************************
     # Assertions should check:
     # - That result is truthy
     # - That tai and jesse's inventories are the correct length
     # - That all the correct items are in tai and jesse's inventories, and that the items that were swapped are not there
-
-@pytest.mark.skip
+    assert result == tai.swap_best_by_category(
+        other=jesse,
+        my_priority="Clothing",
+        their_priority="Decor"
+    )
```

Comment:
`tai.swap_best_by_category(...)` is being called twice, first on line 112 and then here on line 126.  This will result in swapping multiple items, rather than just the first set of items.  The function `swap_best_by_category` returns a boolean True/False value.  This assert should check to see if the result generated by the function call on line 112 is the correct value (in this case True, because this swap should succeed).